options/posix/include/pwd.h

```diff
@@ -16,6 +16,7 @@ struct passwd {
 	gid_t pw_gid;
 	char *pw_dir;
 	char *pw_shell;
+	char *pw_gecos;
```

Comment:
Don't we need to adjust the function that parses `/etc/passwd`?

Reply:
Yes we probably should, I'll add that to the list of things this PR still needs

Reply:
This should be fixed in 6e2b115